tests/util/keyring.py

```diff
@@ -151,7 +151,8 @@ def cleanup(self):
 
         if self.delete_on_cleanup:
             temp_dir = self.keychain._temp_dir
-            print(f"Cleaning up temp keychain in dir: {temp_dir}")
+            # this causes issues with vscode test discovery
+            # print(f"Cleaning up temp keychain in dir: {temp_dir}")
```

Comment:
This has nothing to do with the PR, but this line causes the VS code python extension test module to not work properly

Reply:
I added this to my PR as well. I'll remove it from mine since it appears that yours will be approved first.